### PR TITLE
Manufacturers: Transfer SoG

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15982

*Description of changes:*
* Support conditional access to MHR Transfers for Manufacturers
* Minor text content update for Manufacturers and Qualified Suppliers

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1731-fam69rrf.web.app

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -878,6 +891,11 @@ export default defineComponent({
         await setUnsavedChanges(false)
       }
 
+      // Check for product based Transfer access
+      if(isRoleManufacturer.value) {
+        localState.enableRoleBasedTransfer = await enableManufacturerTransfer()
```
Comment (dimak1) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue:896]: Looks like a future proof for additional conditions?

**Review by dimak1 — APPROVED** [review]
Looks great

**Review by doug-lovett — APPROVED** [review]
Looks fantastico!

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:896]: Exactly! might be a few more coming

**Pull request merged**